AccountingSystem/private/Plotters/BaseReporter.py

```diff
@@ -21,291 +28,374 @@ class BaseReporter( DBUtils ):
   _EA_PADDING = 'figurePadding'
   _EA_TITLE = 'plotTitle'
 
-  _RATE_UNITS = { 'time' : ( ( 'seconds / s', 1, 24 ), ( 'hours / s', 3600, 24 ), ( 'days / s', 86400, 15 ), ( 'weeks / s', 86400 * 7, 10 ), ( 'months / s', 86400 * 30, 12 ), ( 'years / s', 86400 * 365, 1 ) ),
-                  'cpupower' : ( ( 'HS06', 1, 750 ), ( 'kHS06', 1000, 750 ), ( 'MHS06', 10 ** 6, 1 ) ),
-                  'bytes' : ( ( 'MB / s', 10 ** 6, 1000 ), ( 'GB / s', 10 ** 9, 1000 ), ( 'TB / s', 10 ** 12, 1000 ), ( 'PB / s', 10 ** 15, 1 ) ),
-                  'jobs' : ( ( 'jobs / hour', 1 / 3600.0, 1000 ), ( 'kjobs / hour', ( 10 ** 3 ) / 3600.0, 1000 ), ( 'Mjobs / hour', ( 10 ** 6 ) / 3600.0, 1 ) ),
-                  'files' : ( ( 'files / hour', 1 / 3600.0, 1000 ), ( 'kfiles / hour', ( 10 ** 3 ) / 3600.0, 1000 ), ( 'Mfiles / hour', ( 10 ** 6 ) / 3600.0, 1 ) )
-                }
+  _RATE_UNITS = {
+      'time': (
+          ('seconds / s',
+           1,
+           24),
+          ('hours / s',
+           3600,
+           24),
+          ('days / s',
+           86400,
+           15),
+          ('weeks / s',
+           86400 * 7,
+           10),
+          ('months / s',
+           86400 * 30,
+           12),
+          ('years / s',
+           86400 * 365,
+           1)),
+      'cpupower': (
+          ('HS06',
+           1,
+           750),
+          ('kHS06',
+           1000,
+           750),
+          ('MHS06',
+           10 ** 6,
+           1)),
+      'bytes': (
+          ('MB / s',
+           10 ** 6,
+           1000),
```

Comment:
GRRRRRRRRRRRRR here it is power of 10 again. Shall we change that too ?

Reply:
Just tell me what to put... I am a bit lost right now at what's the unit to use :)

Reply:
@chaen ...?

Reply:
Put 1024 here for sizes, probably the easiest. Or use the utility function, but maybe an overkill

Reply:
Shouldn't the powers of 10 be changed to corresponding powers of 2? What does the 1000 represent anyway?